root/var/www/html/config.inc.php

```diff
@@ -4,38 +4,38 @@
  * Database settings
  ******************/
 // Which database system: "pgsql"=PostgreSQL, "mysql"=MySQL
-$dbsys = "mysql";
+$dbsys = $_ENV['MRBS_DB_SYSTEM'];
 // Hostname of database server. For pgsql, can use "" instead of localhost
 // to use Unix Domain Sockets instead of TCP/IP. For mysql "localhost"
 // tells the system to use Unix Domain Sockets, and $db_port will be ignored;
 // if you want to force TCP connection you can use "127.0.0.1".
-$db_host = getenv("DB_HOST");
+$db_host = $_ENV['MRBS_DB_HOST'] ?? $_ENV['DB_HOST'];
 // If you need to use a non standard port for the database connection you
 // can uncomment the following line and specify the port number
 // $db_port = 1234;
 // Database name:
-$db_database = getenv("DB_DATABASE");
+$db_database = $_ENV['MRBS_DB_DATABASE'] ?? $_ENV['DB_DATABASE'];
 // Schema name.  This only applies to PostgreSQL and is only necessary if you have more
 // than one schema in your database and also you are using the same MRBS table names in
 // multiple schemas.
 //$db_schema = "public";
 // Database login user name:
-$db_login = getenv("DB_USER");
+$db_login = $_ENV['MRBS_DB_USER'] ?? $_ENV['DB_USER'];
 // Database login password:
-$db_password = getenv("DB_PASS");
+$db_password = $_ENV['MRBS_DB_PASSWORD']  ?? $_ENV['DB_PASS'];
 // Prefix for table names.  This will allow multiple installations where only
 // one database is available
-$db_tbl_prefix = "mrbs_";
+$db_tbl_prefix = $_ENV['MRBS_DB_TBL_PREFIX'] ?? "mrbs_";
```

Comment:
Making this modifiable by an env variable does not make any sense, since the prefix is hard-coded in the `tables.*.sql`, see:
- https://github.com/meeting-room-booking-system/mrbs-code/blob/c2d5cb8214ca69ff15a639e017882ffea0aa1f26/tables.my.sql#L15
- https://github.com/meeting-room-booking-system/mrbs-code/blob/c2d5cb8214ca69ff15a639e017882ffea0aa1f26/tables.pg.sql#L20